.github/workflows/main.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     - name: Configure shell environment variables
       run: |
         export SWIG_DIR=/c/ProgramData/chocolatey/lib/swig/tools/install/swigwin-3.0.12
-        export JAVA_HOME=/c/hostedtoolcache/windows/jre/8.0.272/x64
+        export JAVA_HOME=`cygpath -u $JAVA_HOME_8_0_275_X64`
```

Comment:
I don't think it will resolve the issue, the version is still hardcoded here (so it will break again when the env is updated)

Reply:
No it won't because we are explicitly asking for Java 1.8 version above. The minor version might change which is what this change is accounting for.

Reply:
I am not sure I understand. What happens when the java package is updated in the runtime environment again? It looks as though you are still asking for a specific java version (which is likely to be updated again in the future)

Reply:
According to https://github.com/actions/setup-java/blob/d202f5dbf7256730fb690ec59f6381650114feb2/README.md
we should also be able to lock java to a specific semver here:
https://github.com/alainmarcel/Surelog/blob/ef488c691c7ff48398b9a4e7eb58a8fa5ea4543a/.github/workflows/main.yml#L82

Maybe we should also change it here to `8.0.275` to make the package doesn't get updated without our knowledge.

Reply:
You are right. Updated the requested version to include the minor as well.